src/tools/ReadyToRun.SuperIlc/ParallelRunner.cs

```diff
@@ -236,7 +268,7 @@ private static void BuildProjects(int startIndex, int endIndex, int totalCount,
                 }
                 while (freeSlot == null);
 
-                freeSlot.Launch(processInfo, jittedMethods, index, totalCount, progressIndex, failureCount);
+                freeSlot.Launch(processInfo, jittedMethods, index + 1, totalCount, progressIndex, failureCount);
```

Comment:
I don't get the purpose of this change - can you please clarify?

Reply:
I'm pretty sure this was a bug (or I have a different idea of the expected behaviour). Before each line it prints `processIndex / processCount: *info*` so that we can tell which process is sending what message, but the processIndex is 0 based so before my change we would get `0 / 1: launching C:\coreclr\...` but I think `1/1` makes more sense. If this was intentional, I could revert it.

Reply:
Well, I don't have any clear preference whether the list should be zero- or one-based, we just need to make sure that the index we print in Launch matches the one in ProcessRunner.IsAvailable because the primary purpose of the indices is to correlate launch and completion of the individual processes in the potentially parallel run. I have double-checked the relevant code and I believe your implementation is correct in this sense.